### ISSUE TITLE
Feature request: GFPGAN for fixing eyes

### ISSUE THREAD

**Issue by tandefelt** [divamgupta/diffusionbee-stable-diffusion-ui#34]
Hi,
A great tool, I agree with all prior feature request, wanted to, respectfully, add one:
GFPGAN utilization, overall, for possible scaling (yes), but.. centrally, for fixing eyes, when creating humans.. 

Naturally, strange eyes can be also very effective, but control over how eyes look sounds like a very welcome addition.. small thing, yet, eyes are windows to the soul.

**Comment by harryshawk** [comment]
Yes -- would be helpful.. My goal is images that can be used in ads, etc. Humans have to look right!

**Comment by jemueller** [comment]
this would be very good yes, thank you

**Comment by trbutler** [comment]
Perhaps the [LStein fork of Stable-Diffusion](https://github.com/lstein/stable-diffusion) offers some help on this? The web UI for it integrates both of these pretty well. Not sure how hard it would be to take its method and move it to a native MacOS app.

**Comment by tandefelt** [comment]
Have to rephrase, once again...    An absolutely amazing tool, thank you so much.. !!! My friends and I here in FInland have gotten into a full "flow" state for days and days, a few nights til the morning.. Anyways, my sincere humble wishlist (based on my 15 years of teaching design/tech/interface design masters thesis at Parsons MFADT: 1) gfpgan, especially eye fixing (eyes are for the most part.. blobby), 2) way higher resolutions (understanding fully the training limit), 3) maybe some super resolution solution (now using topaz tools), 4). faster rendering speeds.. 5) definitely. multiple files outputted once in a batch, once a great result is once gotten, a great option to "cook" more once.. 6) 2D-to-3D/depth map output options.. :) 7) more overall core SD adjustment options.. PS: I saw this one, likely takes some fiddling to get to work with M1/M2: https://replicate.com/cjwbw/stable-diffusion-high-resolution,  anyways: thank you again, super excited about this development, literally blows the door wide open to a whole new world for tons of people. Best regards, Marko, Helsinki, Finland.
